shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumInfoCommand.java

```diff
@@ -69,16 +73,33 @@ public int run(CommandLine cl) throws IOException {
     }
 
     GetQuorumInfoPResponse quorumInfo = jmClient.getQuorumInfo();
+
+    Optional<QuorumServerInfo> leadingMasterInfoOpt = quorumInfo.getServerInfoList().stream()
+            .filter(QuorumServerInfo::getIsLeader).findFirst();
+    if (!leadingMasterInfoOpt.isPresent()) {
+      mPrintStream.println("No leading master was found");
+      return -1;
```

Comment:
Let's just print as such and continue listing members. This might be possible during transitions.

Reply:
Also, I think it'd be consistent style wise if we just append to `OUTPUT_HEADER_LEADING_MASTER` as in regular case. In this case, we can just say "UNKNOWN".

Reply:
fixed